public_website/src/ui/components/simulator/Label.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+
+export const Label = styled.label`
+  display: block;
+  font-size: ${({ theme }) => theme.fonts.sizes['4xl']};
+  font-weight: ${({ theme }) => theme.fonts.weights.bold};
+
+  border-bottom: 1px solid ${({ theme }) => theme.colors.primary}33; // 20% opacity
+  padding-bottom: 2.5rem;
+
+  @media (width < ${({ theme }) => theme.mediaQueries.mobile}) {
+    padding-bottom: 1.5rem;
+  }
+`
```

Comment:
```suggestion
export const Label = styled.label`
  ${({ theme }) => css`
     display: block;
     font-size: theme.fonts.sizes['4xl']};
     font-weight: theme.fonts.weights.bold};

     border-bottom: 1px solid theme.colors.primary; // 20% opacity
     padding-bottom: 2.5rem;

     @media (width < theme.mediaQueries.mobile) {
      padding-bottom: 1.5rem;
  }
`
```

C'est plus lisible comme ça non ? 

Le 33 rajouté a la fin de la couleur c'est pour créer l'opacity ?